app/services/bgs/dependent_service.rb

```diff
@@ -14,6 +14,13 @@ def get_dependents
 
     def submit_686c_form(claim)
       bgs_person = service.people.find_person_by_ptcpnt_id(@user.participant_id)
+
+      # rubocop:disable Rails/DynamicFindBy
+      bgs_person = service.people.find_by_ssn(@user.ssn) if bgs_person.nil?
```

Comment:
I think I left a similar comment on another PR, but does this not work:
```suggestion
      bgs_person = service.people.find_by(ssn: @user.ssn) if bgs_person.nil?
```

Reply:
@f1337 @kathleencrawford and I are on the same team. I wanted to link our conversation here for convenience and so you know we are listening to and value your input.

https://github.com/department-of-veterans-affairs/vets-api/pull/4630#discussion_r470093738